services/web-app/next.config.mjs

```diff
@@ -31,6 +40,9 @@ module.exports = withMDX({
   },
   swcMinify: true,
   webpack(config) {
+    // silence cache warning notifications due to next.config.mjs imports for now, open issues: https://github.com/vercel/next.js/issues/33693 -> https://github.com/webpack/webpack/issues/15574
```

Comment:
maybe this to avoid the eslint max length allowed
```suggestion
// silence cache warning notifications due to next.config.mjs imports for now, open issues:
// https://github.com/vercel/next.js/issues/33693
// https://github.com/webpack/webpack/issues/15574
```